packages/@glimmer/interfaces/lib/managers/modifier.d.ts

```diff
@@ -19,4 +19,6 @@ export interface ModifierManager<ModifierStateBucket> {
   installModifier(instance: ModifierStateBucket, element: Element, args: Arguments): void;
   updateModifier(instance: ModifierStateBucket, args: Arguments): void;
   destroyModifier(instance: ModifierStateBucket, args: Arguments): void;
+  getDebugName?(instance: object): string;
+  getDebugInstance?(instance: object): any;
```

Comment:
`: object` is fishy, probably should be `ModifierStateBucket` or something

Reply:
right